pkg/metrics/state.go

```diff
@@ -0,0 +1,28 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const stateSubsystem = "state"
```

Comment:
Why have you decided to introduce a new hierarchy level and not make it similar to the `epoch` metric?

Reply:
when i did that i thought that it could be extended with the network info (since they are both presented in `control healthcheck` command) in the future, also, i thought about moving `epoch` to that section too as it could be considered as a state. planned to see your thoughts about that

what do you think?

Reply:
Ok, I think moving epoch from the top-level would be nice.